1_Network_Structure.ipynb

```diff
@@ -659,10 +659,10 @@
     "<div class=\"alert-info\">\n",
     "\n",
     "#### Short Answer 3\n",
-    "Write a few sentences about the distance (number of edges) between nodes in this network, considering the following. \n",
+    "Write a few sentences about the distance (number of edges of the shortest path) between nodes in the _A Storm of Swords_ network `G_asoiaf`, considering the following. \n",
```

Comment:
Yes. It is good to indicate the network is for the "_A Storm of Swords_". 
I would not add "of the shortest path" since it duplicates the definition of the distance. 
The "number of edge" was added there to distinguish it from the other option, the "number of intermediate node".